luminoth/tools/dataset/readers/base_reader.py

```diff
@@ -0,0 +1,25 @@
+import abc
+
+
+class InvalidDataDirectory(Exception):
+    """
+    Error raised when the chosen intput directory for the dataset is not valid.
+    """
+
```

Comment:
Why?

Reply:
Because if not, we are relying on the existence of the docstring, which not every class might have.

Adding `pass` to everything would make it more explicit it's supposed to not do anything, vs "there is some missing code here". [Django does it that way](https://github.com/django/django/blob/master/django/core/exceptions.py).